cookbook/networking/fetch-data.md

```diff
@@ -83,9 +83,9 @@ we'll need to:
 ```dart
 Future<Post> fetchPost() async {
   final response = await http.get('https://jsonplaceholder.typicode.com/posts/1');
-  final json = json.decode(response.body); 
+  final _json = json.decode(response.body); 
```

Comment:
To avoid names that are too similar, can we just call the local variable responseJson or some such?

Reply:
Done.